doc/community/developer-guide.md

```diff
@@ -473,13 +473,11 @@ To see the databases available, check out [`src/tests/integration/conftest.py`](
 
 ### Integration tests with cloud databases
 
-We run integration tests against cloud databases like Snowflake, which requires using pre-registered accounts to evaluate their behavior. To initiate these tests, please create a branch in our [ploomber/jupyter repository](https://github.com/ploomber/jupysql).
-
-Please note that if you submit a pull request from a forked repository, the integration testing phase will be skipped because the pre-registered accounts won't be accessible.
+Currently we do not run integration tests against cloud databases like Snowflake and Amazon Redshift.
```

Comment:
add the command showing how to run them locally and the configuration needed

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/982/files#diff-8eaf529f9dc81483bd884039ce78dcea820260cfaf8772a86db1a0de84116518R478-R503)